internal/operands/common-templates/reconcile.go

```diff
@@ -78,6 +84,7 @@ func (c *commonTemplates) RequiredCrds() []string {
 }
 
 func (c *commonTemplates) Reconcile(request *common.Request) ([]common.ReconcileResult, error) {
+	c.manageSingleReplicaInfrastructure(request)
```

Comment:
can you move this fn to here https://github.com/kubevirt/ssp-operator/blob/master/internal/operands/common-templates/reconcile.go#L51, because it doesn't have to run every reconcile iteration, does it?

Reply:
I think that you are right, we can move this in creation. In this case `setup.go` should pass `infrastructureTopology` and `scheme|codec` information. Thanks for your review

Reply:
Or you can move it to this file: https://github.com/kubevirt/ssp-operator/blob/master/internal/template-bundle/bundle.go
It contains code that loads the templates from yaml.

Reply:
Agreed, thanks :+1: